benchmarks/perception/a3_stereo_image_proc/benchmark.yaml

```diff
@@ -0,0 +1,85 @@
+id: a3
```

Comment:
One advantage of markdown over yaml in this context would be that GitHub beautifies markdown rendered in the browser.  Is there a similar beautifier for yaml?  How can the contents of this file be easily parsed by a reader/user?

Reply:
We've got tools that convert `yaml` into Markdown automatically. See https://github.com/robotperf/benchmarks/blob/main/benchmarks/other/ros2benchmark/ros2benchmark/verb/update.py.

This can be invoked as:

```bash
ros2 benchmark update
```

and should run in simple CI job in this repo after each commit. That way we keep `yaml` and `README.md` files consistent. For archiving/databases purposes, I'd vote for yaml or similar. Markdown's worse in my experience doing that (and actually not meant for it, though of course you could hack it for that).